cpp/pybind/core/linalg.cpp

```diff
@@ -50,7 +51,18 @@ void pybind_core_linalg(py::module &m) {
             "Function to perform matrix multiplication of two 2D tensors with "
             "compatible shapes.",
             "A"_a, "B"_a);
-
+    m.def(
+            "addmm",
+            [](const Tensor &input, const Tensor &A, const Tensor &B,
+               double alpha, double beta) {
+                Tensor output =
+                        input.Expand({A.GetShape(0), B.GetShape(1)}).Clone();
+                AddMM(A, B, output, alpha, beta);
+                return output;
+            },
+            "Function to perform addmm of two 2D tensors with compatible "
```

Comment:
mention the operation performed. ref. https://pytorch.org/docs/stable/generated/torch.addmm.html